paddle/phi/api/yaml/op_compat.yaml

```
@@ -818,6 +818,12 @@
   outputs :
     out : Out
 
+- op : dgc_momentum
+  inputs :
+    {param : Param, grad : Grad, velocity : Velocity, learning_rate : LearningRate, master_param : MasterParam, float mu, bool use_nesterov, str regularization_method, float regularization_coeff, bool multi_precision, float rescale_grad, float rampup_begin_step}
```

```suggestion
    {param : Param, grad : Grad, velocity : Velocity, learning_rate : LearningRate, master_param : MasterParam}
```
op_compat里面只声明必要的映射就可以了，后面的没必要复制粘贴